### PR TITLE
fix(disabled): fixed styles to prevent clear while disabled #221

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #221

## Summary:

Update styles so that "clear" cannot be clicked while the input is disabled.

Also insure the cursor does not change while hovering over the space where the clear button is while the input is disabled.

Cleanup other linter issues with styles.

Update deps to latest versions.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jordanjones243 — COMMENTED** [review]
I tested the code and everything looks good. The one thing I wonder about is if the input cursor icon is supposed to show on hover. If so, everything looks great and I will approve the PR.

**Comment by jason-capsule42** [comment]
> I tested the code and everything looks good. The one thing I wonder about is if the input cursor icon is supposed to show on hover. If so, everything looks great and I will approve the PR.

Good callout. Added a small CSS update to account for that.

File: package.json [hunk 1]
```diff
@@ -32,57 +32,57 @@
...
-    "concurrently": "^6.4.0",
+    "concurrently": "^7.6.0",
     "copyfiles": "^2.4.1",
-    "core-js": "^3.19.1",
-    "eslint": "^8.3.0",
+    "core-js": "^3.27.1",
+    "eslint": "^8.32.0",
     "eslint-plugin-jsdoc": "^37.0.3",
-    "husky": "^7.0.0",
+    "husky": "^8.0.3",
```
Comment (blackfalcon) [inline package.json:61]: Good update!

**Pull request merged**